scripts/vis_trawler.py

```diff
@@ -310,7 +309,7 @@ def list_trawl_files(prod_dir, file_match, file_writing, complete_token, time_ou
     complete = False
     # check for failed token, if there return an empty list and incomplete.
     if os.path.isfile(os.path.join(prod_dir, "failed")):
-        logger.warning("%s so not processing, moving to failed directory." % (os.path.join(prod_dir, "failed")))
+        logger.warning("%s so not processing, moving to failed directory.", os.path.join(prod_dir, "failed"))
```

Comment:
I wonder if we should try and include some indication of the failure type, so we don't have to go and look at the failed file to find out what went wrong.

Reply:
I've added:
`logger.exception("Caught exception while extracting metadata from %s.", err.filename) `